ksqldb-functional-tests/src/test/resources/query-validation-tests/row-offset-and-partition.json

```diff
@@ -80,34 +80,65 @@
       }
     },
     {
+      "name": "verify ROWPARTITION is not queryable with feature disabled",
+      "properties": {
+        "ksql.rowpartition.rowoffset.enabled": false
+      },
+      "statements": [
+        "CREATE STREAM input (id INT KEY, val INTEGER) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM output AS SELECT ROWPARTITION FROM input EMIT CHANGES;"
+      ],
+      "expectedException": {
+        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "message": "SELECT column 'ROWPARTITION' cannot be resolved."
+      }
+    },
+    {
+      "name": "verify ROWOFFSET is not queryable with feature disabled",
+      "properties": {
+        "ksql.rowpartition.rowoffset.enabled": false
+      },
+      "statements": [
+        "CREATE STREAM input (id INT KEY, val INTEGER) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM output AS SELECT ROWOFFSET FROM input EMIT CHANGES;"
+      ],
+      "expectedException": {
+        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "message": "SELECT column 'ROWOFFSET' cannot be resolved."
+      }
+    },
+    {
+      "comments": "Should be removed after rowpartition.rowoffset.enabled feature flag is removed",
```

Comment:
I don't think this test (and the next three) should be removed after the feature flag is removed.

Reply:
I thought the idea is that we change the non `feature-enabled` flags to copy this behavior, then remove these tests?

https://github.com/confluentinc/ksql/pull/8120#discussion_r712298315

Reply:
Oh I see. I didn't realize there was another copy of this test. In that case we need to remove the comments that say to remove the other copy, else we've got comments saying that both copies of the test should be removed: https://github.com/confluentinc/ksql/blob/e8d5132c5503ec25d579082397c67d7d24d6fcba/ksqldb-functional-tests/src/test/resources/query-validation-tests/row-offset-and-partition.json#L7

Reply:
I think I removed these comments in the still-unmerged pull query PR. I removed said removal of the comments when merging in master to that branch today so I could put the changes on this branch, so this will indeed be necessary to do again.